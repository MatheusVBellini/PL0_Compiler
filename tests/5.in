VAR a, b;
BEGIN
  a := 5;
  b := a + 10;
END.
