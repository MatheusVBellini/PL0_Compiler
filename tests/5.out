VAR, VAR
a, ident
,, simbolo_virgula
b, ident
;, simbolo_ponto_virgula
BEGIN, BEGIN
a, ident
:=, simbolo_atribuicao
5, numero
;, simbolo_ponto_virgula
b, ident
:=, simbolo_atribuicao
a, ident
+, simbolo_mais
10, numero
;, simbolo_ponto_virgula
END, END
., simbolo_ponto
